packages/edit-post/src/components/header/more-menu/index.js

```diff
@@ -17,26 +20,18 @@ import WritingMenu from '../writing-menu';
 
 const POPOVER_PROPS = {
 	className: 'edit-post-more-menu__content',
-	position: 'bottom left',
-};
-const TOGGLE_PROPS = {
-	tooltipPosition: 'bottom',
 };
 
 const MoreMenu = ( { showIconLabels } ) => {
 	const isLargeViewport = useViewportMatch( 'large' );
 
 	return (
-		<DropdownMenu
+		<MoreMenuDropdown
```

Comment:
Nice one 👍🏻

Reply:
That was part of this PR - https://github.com/WordPress/gutenberg/pull/34154.

I had to rebase this one after it was merged so that the diff was right.

The last editor that require a More Menu update is the site editor, which is in this PR - https://github.com/WordPress/gutenberg/pull/34191